nft/Cargo.toml

```diff
@@ -8,5 +8,5 @@ edition = "2018"
 crate-type = ["cdylib", "rlib"]
 
 [dependencies]
-near-sdk = "3.1.0"
-near-contract-standards = "3.1.1"
+near-sdk = "4.0.0"
```

Comment:
You'll also need to update `workspaces-rs` to version `0.3` to support the newest SDK features. @volovyk-s is updating `workspaces-js` to support the latest SDK changes so we'll have to wait until then for the JS side of tests